### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by Mutalenic**
This captstone project is about a HIR conference.
I built this project using the following;

- Used HTML ,CSS and JavaScript
- Used flexbox and grid
- Made the project for both mobile and desktop
- Used Javascript to manipulate DOM elements and events
- Deployed the project at  https://mutalenic.github.io/HIR-Africa/
- Made a video presentation  - https://www.loom.com/share/323023b0908e445e87ba893d49237034

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+const speakerCard = document.querySelector('.SpeakerSection');
+
+const speakersinfo = [
+  {
+    Name: 'Mwiya Musokotwane',
+    Picture: './images/Mwiya1.jpg',
+    ImageAlt: 'Mwiya Musokotwane',
+    speakerTitle: 'CEO Thebe Investment',
+    Bio: 'Mwiya Musokotwane is the CEO of Zambia-based investment firm Thebe Investment Management.',
```
Comment (Amrhub) [inline main.js:29]: image
- [ ] I noticed images are working in our local but it fails when deployed, Let's make sure we fix that.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    <p class="logoText"> The logo of HIR Africa Conference 2022 was decided through the logo
+        competition last year from 20 May to 28 May.</p>
+        <div><img src="Images/Hirlogo2.jpg" class="logoImage" alt="logo image" ></div>
+        <hr class="sectionDividerLine">
+</section>
+<section class="pastEvents">  
+<h3>See the past HIR Africa Conference</h3>
+<p class="pastText">Take a look at the past HIR Africa conference which took place in
+    Ghana and Rwanda</p>
+    <div class="pastEventImage">
```
Comment (Amrhub) [inline about.html:84]: - [ ] I also noticed the event images were working locally, but not working anymore when deployed let's make sure we fix that.
This is how it looks now: 
image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Mutalenic,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

These changes aim to add some awesomeness to your awesome project and code skills.



## To Highlight ✔️ 
Clean and Organized code 👏🏻 
Used correct workflow and linters are passing ✔️ 
Documentation of the project in a professional way ✔️
Recorded project overview ✔️ 
## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] I would recommend practicing more in front of the camera what you want to say in the video before recording that will help you talk at a normal pace, be less nervous, and focus on good points, I used to follow this technique it worked for me :shipit:.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (Mutalenic) [reply main.js:29]: Fixed,thanks

File: about.html (hunk 2 above)
Comment (Mutalenic) [reply about.html:84]: Fixed,thanks

**Pull request merged**